tests/manage/mcg/test_bucket_creation_deletion.py

```diff
@@ -148,14 +148,37 @@ class TestBucketCreation(MCGTest):
             "1-CLI-PVPOOL",
         ],
     )
-    def test_bucket_creation(
-        self, bucket_class_factory, bucket_factory, amount, interface, bucketclass_dict
+    def test_bucket_creation_deletion(
+        self,
+        verify_rgw_restart_count,
+        mcg_obj,
+        bucket_class_factory,
+        bucket_factory,
+        amount,
+        interface,
+        bucketclass_dict,
     ):
         """
         Test bucket creation using the S3 SDK, OC command or MCG CLI.
         The factory checks the bucket's health by default.
         """
-        bucket_factory(amount, interface, bucketclass=bucketclass_dict)
+        buckets = bucket_factory(amount, interface, bucketclass=bucketclass_dict)
+
+        if amount == 3 and interface == "OC":
+            unexpected_log = 'malformed BucketHost "s3.openshift-storage.svc": malformed subdomain name "s3"'
+            rook_op_pod = get_operator_pods()
+            pod_log = get_pod_logs(pod_name=rook_op_pod[0].name)
+            assert not (
+                unexpected_log in pod_log
+            ), f"Bucket notification errors found {unexpected_log}"
+
```

Comment:
It is great to see that you have combined test_check_for_bucket_notification_error_in_rook_op_logs here. With this change, this test will be run as part of the acceptance run. I feel this log check test is not a candidate for the acceptance suite and this change could potentially increase the overall run time of the acceptance suite (though the increase will be very small). Can we limit the execution of this log check to only tier 1, ignoring acceptance?

Reply:
addressed